smartsim/settings/mpirunSettings.py

```diff
@@ -134,6 +137,61 @@ def set_hostlist(self, host_list):
             raise TypeError("host_list argument must be list of strings")
         self.run_args["host"] = ",".join(host_list)
 
+    def set_verbose_launch(self, verbose):
+        """Set the job to run in verbose mode
+
+        This sets ``--verbose``
+
+        :param verbose: Whether the job should be run verbosely
+        :type verbose: bool
+        """
+        if verbose:
+            self.run_args["verbose"] = None
+            return
+        if "verbose" in self.run_args:
+            del self.run_args["verbose"]
+
+    def set_quiet_launch(self, quiet):
+        """Set the job to run in quiet mode
+
+        This sets ``--quiet``
+
+        :param quiet: Whether the job should be run quietly
+        :type quiet: bool 
+        """
+        if quiet:
+            self.run_args["quiet"] = None
+            return
+        if "quiet" in self.run_args:
+            del self.run_args["quiet"]
+
+    def set_broadcast(self, dest_path=None):
+        """Copy the specified executable(s) to remote machines
+
+        This sets ``--preload-binary``
+
+        :param dest_path: Destination path (Ignored)
+        :type dest_path: str | None
+        """
+        if dest_path is not None and isinstance(dest_path, str):
+            logger.warning(
+                (
+                    f"{type(self)} cannot set a destination path during broadcast. "
+                    "Using session directory instead"
+                )
+            )
+        self.run_args["preload-binary"] = None
+
+    def set_timeout(self, time):
+        """Set the maximum number of seconds that a job will run
+
+        This sets ``--timeout``
+
+        :param time: The maximum number of seconds that a job will run in secs
+        :type quiet: int
+        """
+        self.run_args["timeout"] = int(time)
```

Comment:
on these methods that supply a time constraint, I feel like we can improve upon "number of seconds". 

for example, what about ``hours=1, minutes=0, seconds=0`` as the default and we just convert to seconds. I feel like we could do this in other places as well.

Reply:
Like the idea!! My only concern would be defaulting the `hours` param to be 1. I think it seems a bit un-intuitive that that:
```py
rs: RunSettings = ...
rs.set_time(seconds=5)
```
would set the time limit to 365 seconds from now. We could do scan the parameters and manually correct to the time to be 5 seconds seconds in the future, but I think the default being 0 / unlimited time makes more sense.

Reply:
Basic implementation of `set_time` in [b9aec16](https://github.com/CrayLabs/SmartSim/pull/170/commits/b9aec16118b55f342a31d4b3e77b8e60743ab137)